extensions/amp-story/1.0/amp-story-hint.css

```diff
@@ -86,15 +85,15 @@
 
 .i-amphtml-story-hint-container .next-page {
   flex: 3 !important;
-  border-left: 1px dashed transparent !important;
-  border-image-source: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" version="1.1"><line x1="0" y1="0" x2="0" y2="30" fill-rule="evenodd" stroke-width="1" fill="none" stroke-linecap="square" stroke-dasharray="6,6" stroke="white"></line></svg>')  !important;
-  border-image-slice: 33% 33%  !important;
-  border-image-repeat: repeat  !important;
-  border-image-width: 14px !important;
+  background-image: linear-gradient(#e0d5d5 60%, rgba(255,255,255,0) 0%) !important;
```

Comment:
Where does the `#e0d5d5` come from?
Are we updating the design with this PR?

Reply:
> Where does the #e0d5d5 come from?

/shrug. It's as close as I could get with my designer lens to try and replicate the svg. Let me know if you want to use a specific value.

Here's the before and after

Before (svg)
<img width="336" alt="svg before" src="https://user-images.githubusercontent.com/5449100/62798138-6391c580-ba92-11e9-8d4c-5381330b6193.png">

After (background repeat)
<img width="337" alt="backround repeat" src="https://user-images.githubusercontent.com/5449100/62798147-668cb600-ba92-11e9-8d16-aaca29bff384.png">

Reply:
Oh these are the dashes, got it! Thanks :))